.gitignore

```diff
@@ -57,7 +57,6 @@ BenchmarkDotNet.Artifacts/
 project.lock.json
 project.fragment.lock.json
 artifacts/
-**/Properties/launchSettings.json
```

Comment:
We could consider keeping this here, and just forcibly adding the launchSettings.json file you are adding which will track it anyway.

Reply:
I wasn't sure whether this file should be generally commited, but I guess you're right.